core/templates/domain/suggestion/suggestion.model.ts

```diff
@@ -24,6 +24,7 @@ interface SuggestionChangeBackendDict {
   'state_name': string;
   'new_value': SuggestionChangeValue;
   'old_value': SuggestionChangeValue;
+  'skill_id'?: string;
```

Comment:
Why `?`?

Reply:
? removed in PR https://github.com/oppia/oppia/pull/15859, PTAL